### DISCUSSION TITLE
[QUESTION] Default style for active and non-active tabs

### DISCUSSION THREAD

**Discussion by open-ssl** [GrapesJS/grapesjs#4809] [Q&A]
Hey! 
I am doing my own custom tabs for grapesjs. I have 2 nonactive tabs and 1 active all of the time.  (1 picture)
<img width="1018" alt="image" src="https://user-images.githubusercontent.com/39589479/209653696-f09cef61-52cb-4d3e-8006-042e1adfda06.png">

What is the default flow in grapesjs to change style for nonactive and active elements in [that repo](https://github.com/artf/grapesjs-tabs) ?

Another one question. I moved all that logic in traits section. (2 picture)
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/39589479/209653952-61b24915-f2bd-4399-8aff-c156b9cb728f.png"> 

Now I gonna get my active and non active class styles by click of button. 

Inside of trait style button handler i have code like that: 

```
{
      full: 1,
      type: 'button',
      label: false,
      text: 'Style for Active Tab',
      command: (ed) => {
        const openSm = ed.Panels.getButton('views', 'open-sm');
        openSm && openSm.set('active', 1);
        const cls = `${config.classTabActiveArea}`;
        ???? ed.AssetManager.setTarget(cls, { targetIsClass: 1 }) ??? 
      },
    },
```

In section with '???' i gonna set active or passsive class of my element in Style Manager. 
There's no info in docs about the way for swapping active class for element. 

So, the question is: 'How can I change active class for Style Manager using editor object?'

Any solutions?

**Comment by artf** [comment]
Try to use this API: `editor.Selectors.select('.my-class')`
